### ISSUE TITLE
cargo bench: zopfli benchmarks hangs?

### ISSUE THREAD

**Issue by matthiaskrgr** [shssoichiro/oxipng#131]
````
     Running target/release/deps/zopfli-2c57822585201135
running 5 tests
````
The benchmark was not progressing for 1.5 hours.

**Comment by RReverser** [comment]
It doesn't hang, it just takes realllllyyy long time due to how slow zopfli is and how many iterations cargo has to make to get reasonably precise results. I left it overnight and it has finished successfully.

**Comment by shssoichiro** [comment]
Yeah, this probably does require some tuning, and maybe moving to something like Criterion for benchmarks would also help here.

**Comment by andrews05** [comment]
This should no longer be an issue:
- The zopfli test file is now much smaller.
- Zopfli is now faster.
